ee/dtr/user/manage-images/sign-images/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Sign an image
+title: Using Notary to sign an image
```

Comment:
Again, just to be clear "singing with notary" historically implies signing with the notary cli.  This instructions are for signing with "docker trust" commands in the docker CLI, which in theory is easier to understand.  It's notary under the hood, but it might be good to highlight that the notary client isn't required anymore.